packages/core/test/invoke.test.ts

```diff
@@ -53,7 +53,8 @@ const fetchMachine = createMachine({
     failure: {
       entry: sendParent({ type: 'REJECT' })
     }
-  }
+  },
+  output: ({ event }) => event.output
```

Comment:
I'm baffled by this - the tests in https://github.com/statelyai/xstate/pull/4306 pass even without it so it indicates that something is wrong.

Reply:
I have no idea what that test would supposed to test 😅 I restructured it slightly and removed `output` from here completely - since the test wasn't testing it anyway.